sdk/resourcemanager/azure-resourcemanager-resources/src/main/java/com/azure/resourcemanager/resources/fluentcore/utils/SdkContext.java

```diff
@@ -15,9 +15,9 @@
  * The class to contain the common factory methods required for SDK framework.
  */
 public class SdkContext {
-    private Function<String, IdentifierProvider> identifierFunction = name -> new ResourceNamer(name);
+    private Function<String, IdentifierProvider> identifierFunction = ResourceNamer::new;
     private static DelayProvider delayProvider = new ResourceDelayProvider();
-    private static Scheduler reactorScheduler = Schedulers.boundedElastic();
+    private static Scheduler reactorScheduler = Schedulers.parallel();
```

Comment:
Previously it is RxJava `Scheduler.io()` http://reactivex.io/RxJava/2.x/javadoc/io/reactivex/schedulers/Schedulers.html#io--

Reactor no long have this.

Since now azure-core is all async without blocking, I think we can change this to default `parallel` Schedular.